src/services/anchor-service.ts

```diff
@@ -225,17 +227,29 @@ export class AnchorService {
 
     // Update the database to record the successful anchors
     logger.debug('Persisting results to local database')
-    const numAnchoredRequests = await this._persistAnchorResult(anchors, candidates)
+    const persistedAnchorsCount = await this._persistAnchorResult(anchors, candidates)
```

Comment:
its getting a little hard to keep track of the difference between `anchorCount`, `anchoredRequestCount`, and `reAnchoredCount` (and how they correspond to `anchors` and `anchoredRequests`).  Could we add some comments to make that clearer?